client/packages/system/src/IndicatorsDemographics/DetailView/utils.tsx

```diff
@@ -77,3 +83,38 @@ export const calculateAcrossRow = (
 
   return updatedRow;
 };
+
+export const getDefaultHeaderData = (baseYear: number): HeaderData => ({
+  baseYear,
+  id: '',
+  '1': { id: '1', value: 1 },
+  '2': { id: '2', value: 1 },
+  '3': { id: '3', value: 1 },
+  '4': { id: '4', value: 1 },
+  '5': { id: '5', value: 1 },
+});
```

Comment:
This is confusing with you also returning a default projection from the query hook if the list is empty, especially because there you return 0s and here you return 1s... can we pick one? 🤷‍♀️

Reply:
I'll remove this and see what happens. basically changed direction mid way and kept this in.